vendor/github.com/google/cel-go/cel/options.go

```diff
@@ -57,6 +56,11 @@ const (
 	// Enable eager validation of declarations to ensure that Env values created
 	// with `Extend` inherit a validated list of declarations from the parent Env.
 	featureEagerlyValidateDeclarations
+
+	// Enable the use of the default UTC timezone when a timezone is not specified
+	// on a CEL timestamp operation. This fixes the scenario where the input time
+	// is not already in UTC.
+	featureDefaultUTCTimeZone
```

Comment:
should we enable this feature before going to beta?

Reply:
Yes we will. Currently waiting for a bug fix from cel before enabling this feature